apisix/cli/ops.lua

```diff
@@ -450,20 +450,20 @@ local function start(env, ...)
     local parser = argparse()
     parser:argument("_", "Placeholder")
     parser:option("-c --config", "location of customized config.yaml")
+    parser:flag("--verbose", "show init_etcd debug information")
```

Comment:
I think `--verbose` can be designed not only for `init_etcd` but let APISIX cli works in verbose mode. So maybe "verbose mode" in description is better.

Reply:
That is indeed a good idea I think. But I go through APISIX cli and for me I could not find many places to add more log. Maybe we could leave it here, and wait if anyone need to add some log.

Reply:
OK, got you.